File: app/src/main/java/com/example/lizardstock/interfaces/IDetailProduct.java
package com.example.lizardstock.interfaces;

import android.net.Uri;

public interface IDetailProduct {

    interface View{
        void successMessage(Boolean success);
    }

    interface Presenter{
        void successMessage(Boolean success);
        void firebaseUpdate(final String codigo, final String nombre, final String cantidad,
                            final String precio, final String categoria, final Uri imagenUri);
    }

    interface Interactor{
        void firebaseUpdate(final String codigo, final String nombre, final String cantidad,
                            final String precio, final String categoria, final Uri imagenUri);
    }

}

File: app/src/main/java/com/example/lizardstock/interfaces/IDetailProduct.kt
package com.example.lizardstock.interfaces

interface IDetailProduct {
    interface View {
        fun successMessage(success: Boolean)
    }

    interface Presenter {
        fun successMessage(success: Boolean)
        fun firebaseUpdate(codigo: String, nombre: String, cantidad: String,
                           precio: String, categoria: String)
    }

    interface Interactor {
        fun firebaseUpdate(codigo: String, nombre: String, cantidad: String,
                           precio: String, categoria: String)
    }
}